### DISCUSSION TITLE
how to make the modal not close when clicking away?

### DISCUSSION THREAD

**Discussion by alessandroprudencio** [GrapesJS/grapesjs#3839] [Q&A] [answered]
I tried to use these attributes but without success:

```
modal.open({
    title: 'Settings',
    width: 380,
    custom: true,
    backdrop: false,
    height: 500,
    attributes: {
      id: 'id-settings',
      'data-backdrop': "false",
      class: 'modal-settings'
    },
  });

```
how to make the modal not close when clicking away?

**Comment by artf** [comment, marked as answer]
The default Modal doesn't have such a feature, but you can connect any other modal library if you need https://grapesjs.com/docs/modules/Modal.html#custom-modal

**Comment by DhanaDSP1120** [comment]
@alessandroprudencio , **Avoid Auto-close of Modal** while clicking away 
`grapesjs.init({
            container: "#grapesjsEditor",
            width: "100%",
            modal: {
            backdrop: false,
            },
})`
